docs/userguide/failover/application-failover.md

```diff
@@ -216,11 +218,14 @@ spec:
       purgeMode: Immediately
       statePreservation:
         rules:
-          - aliasLabelName: pre-updated-replicas
-            jsonPath: "{ .updatedReplicas }"
+          - aliasLabelName: application.karmada.io/failover-jobid
+            jsonPath: "{ .jobStatus.jobID }"
 ```
 
-The above configuration will parse the `updatedReplicas` field from the application `.status` before migration. Upon successful migration, the extracted data is then re-injected into the new resource, ensuring that the application can resume operation with its previous state intact.
+1. Before migration, the Karmada controller will extract the job IDs according to the `jsonPath` configured by the user.  
+2. During migration, the Karmada controller injects the extracted job ID into the Flink application configuration as a label, e.g. `application.karmada.io/failover-jobid : <jobID>`. 
+3. Kyverno running on a member cluster intercepts the Flink application creation request and gets the checkpoint data storage path for the job based on the `jobID`, e.g. `/<shared-path>/<job-namespace>/<jobId>/checkpoints/xxx`, then configures the `initialSavepointPath` to indicate that it will start from the save point.  
```

Comment:
```suggestion
3. Kyverno running on a member cluster intercepts the FlinkDeployment creation request and gets the checkpoint data storage path for the job based on the `jobID`, e.g. `/<shared-path>/<job-namespace>/<jobId>/checkpoints/xxx`, then configures the `initialSavepointPath` to indicate that it will start from the save point.  
```